### PR TITLE
add cattle-cluster-agent debug logs

### PR DESCRIPTION
**Pull request by felipe-colussi**
Fixes: There is no issue that this fix.

It adds documentation for the fix created on this issue: https://github.com/rancher/rancher/issues/42241

## Description

Add to the logs doc the possibility of using Environment Variables to get logs out of the cattle-cluster-agent.

### CODE REVIEW

File: docs/troubleshooting/other-troubleshooting-tips/logging.md [hunk 1]
```diff
@@ -78,3 +78,23 @@ possible to enable debug logs in `rancher-machine` when provisioning RKE1, RKE2
...
+the log level.  
+
+
+For trace-level `CATTLE_TRACE` or `RANCHER_TRACE` must be set to `true`
+
+For debug-level `CATTLE_DEBUG` or `RANCHER_DEUBG` must be set to `true`
+
+:::caution
+
+Just like the `trace` log level above, `cattle-cluster-agent` debug logs can contain sensitive information.
```
Comment (martyav) [inline docs/troubleshooting/other-troubleshooting-tips/logging.md:98]:
```suggestion
The `cattle-cluster-agent` debug logs may contain sensitive information.
```

File: docs/troubleshooting/other-troubleshooting-tips/logging.md [hunk 2]
```diff
@@ -78,3 +78,23 @@ possible to enable debug logs in `rancher-machine` when provisioning RKE1, RKE2
 Just like the `trace` log level above, `rancher-machine` debug logs can contain sensitive information.
 
 :::
+
+
+## Cattle-cluster-agent debug logs
+
+If you need to troubleshoot the initialization of `cattle-cluster-agent` the log levels can be set during the 
+initialization of the downstream clusters.
```
Comment (martyav) [inline docs/troubleshooting/other-troubleshooting-tips/logging.md:86]:
```suggestion
The `cattle-cluster-agent` log levels can be set when you initialize the downstream clusters.  
```

File: docs/troubleshooting/other-troubleshooting-tips/logging.md [hunk 3]
```diff
@@ -78,3 +78,23 @@ possible to enable debug logs in `rancher-machine` when provisioning RKE1, RKE2
...
 :::
+
+
+## Cattle-cluster-agent debug logs
+
+If you need to troubleshoot the initialization of `cattle-cluster-agent` the log levels can be set during the 
+initialization of the downstream clusters.  
+
+Wile creating a Cluster under `Cluster Configuration -> Agent Environment Vars` some variables can be set to define
+the log level.
```
Comment (martyav) [inline docs/troubleshooting/other-troubleshooting-tips/logging.md:89]:
```suggestion
When you create a cluster under **Cluster Configuration > Agent Environment Vars** you can set variables to define the log level.  
```

File: docs/troubleshooting/other-troubleshooting-tips/logging.md [hunk 4]
```diff
@@ -78,3 +78,23 @@ possible to enable debug logs in `rancher-machine` when provisioning RKE1, RKE2
...
+If you need to troubleshoot the initialization of `cattle-cluster-agent` the log levels can be set during the 
+initialization of the downstream clusters.  
+
+Wile creating a Cluster under `Cluster Configuration -> Agent Environment Vars` some variables can be set to define
+the log level.  
+
+
+For trace-level `CATTLE_TRACE` or `RANCHER_TRACE` must be set to `true`
+
+For debug-level `CATTLE_DEBUG` or `RANCHER_DEUBG` must be set to `true`
```
Comment (martyav) [inline docs/troubleshooting/other-troubleshooting-tips/logging.md:94]:
```suggestion
- Trace-level logging: Set `CATTLE_TRACE` or `RANCHER_TRACE` to `true`

-  Debug-level logging: Set `CATTLE_DEBUG` or `RANCHER_DEUBG` to `true`
```

**Review by martyav — CHANGES_REQUESTED** [review]

**Comment by a-blender** [comment]
Trying to merge unnecessary commits to wrong branch, @felipe-colussi will open a new PR.

**Pull request closed without merging**